lib/active_model/serializer/concerns/configuration.rb

```diff
@@ -31,6 +31,28 @@ def config.array_serializer
         # ref: http://jsonapi.org/format/#document-top-level
         config.jsonapi_include_toplevel_object = false
 
+        # example output
+        # => [
+        # "::Api::V1::ResNamespace::ResourceSerializer",
+        # "::Api::ResNamespace::ResourceSerializer",
+        # "::ResNamespace::ResourceSerializer" ,
+        # "::ResourceSerializer"]
+        config.serializer_lookup = proc { |controller_name|
+          lookups = []
+          current_resource = controller_name.gsub(/Controller\z/, '')
```

Comment:
that's true. I'll switch it to lookup_namespace, and mention in a comment that it *can* and often is the controller name, cause of https://github.com/rails-api/active_model_serializers/pull/1757/files#diff-5624f9cf35be4dc254412d979fd293c4R63 klass is a controller (I guess, most of the time)